.github/workflows/PR.yml

```diff
@@ -10,8 +10,8 @@ jobs:
     steps:
       # actions/checkout@v2 but we use the SHA1 because tags can be re-written in git
       - uses: actions/checkout@722adc63f1aa60a57ec37892e133b1d319cae598
-      - name: Select Xcode 12.5
-        run: sudo xcode-select -s /Applications/Xcode_12.5.app
+      - name: Select Xcode 13.0
+        run: sudo xcode-select -s /Applications/Xcode_13.0_beta.app
```

Comment:
According to [the macOS 11 README](https://github.com/actions/virtual-environments/blob/main/images/macos/macos-11-Readme.md#xcode) there is no `_beta` in the path, it should just be `/Applications/Xcode_13.0.app` and it'll pick up the latest beta (I think).

Reply:
```suggestion
        run: sudo xcode-select -s /Applications/Xcode_13.0.app
```